### DISCUSSION TITLE
Can we force using rebase instead of merge ?

### DISCUSSION THREAD

**Discussion by pierresisson** [NostrGit/NostrGit#104] [General]
Hello there,

can we considerate using `rebase` instead of `merge` ?
context: https://www.atlassian.com/git/tutorials/merging-vs-rebasing

i love using it because it keeps the history of `main` branch.
the githubs settings provide a params where when can choose to update branch with rebase
![image](https://user-images.githubusercontent.com/79455860/223554503-8b865d56-e761-418a-8029-7b3d02e4e405.png)

btw we can also use `squash and merge`, only pull request `commit` matters

What do you think?

**Comment by cypherhoodlum** [comment]
We [decided](https://github.com/NostrGit/NostrGit/discussions/95) to not use Discussions anymore. Let's continue the discussion at #117  :heart:
